helm 5.4.0: add release notes (#8492)
grafana/mimir: docs/sources/helm-charts/mimir-distributed/release-notes/v5.4.md

---
title: "Grafana Mimir Helm chart version 5.4 release notes"
menuTitle: "V5.4 release notes"
description: "Release notes for Grafana Mimir Helm chart version 5.4"
weight: 300
---

# Grafana Mimir Helm chart version 5.4 release notes

Grafana Labs is excited to announce version 5.4 of the Grafana Mimir Helm chart, which is compatible with Grafana Mimir v2.13 or Grafana Enterprise Metrics (GEM) v2.13. The `mimir-distributed` Helm chart is the best way to install Mimir on Kubernetes.

The highlights that follow include the top features, enhancements, and bug fixes in this release. For a comprehensive list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).

## Important changes

> A number of timeout settings have been improved upon:

Reviewer: ```suggestion
A number of timeout settings are improved:
```

> - Fine-tuned termination grace periods for all components. This helps with ensuring that components can shut down gracefully without interrupting queries or write requests.

Reviewer: ```suggestion
- Fine-tuned termination grace periods for all components. This helps to ensure that components can shut down gracefully, without interrupting queries or write requests.
```

> - Continuous test and smoke test containers now use the Mimir image as now the continuous test code is part of Mimir.

Reviewer: ```suggestion
- Continuous test and smoke test containers now use the Mimir image, as the continuous test code is now part of Mimir.
```

>   As a result, the `image` section in the values file now replaces `smoke_test.image` and `continuous_test.image` and the latter two have been removed.

Reviewer: ```suggestion
  As a result, the `image` section in the values file now replaces `smoke_test.image` and `continuous_test.image`, and the latter two are removed.
```

## Features and enhancements

Notable enhancements are as follows:

- Experimental support for a dedicated ruler query path. With a dedicated set of queriers, query-frontends, and

>   query-schedulers outages and slow-downs on the main query path do not affect rule evaluations.

Reviewer: ```suggestion
  query-schedulers, outages and slow-downs on the main query path don't affect rule evaluations.
```
